src/classes/REL_Relationships_TEST.cls

```diff
@@ -226,74 +217,105 @@ public class REL_Relationships_TEST {
     public static void testAutoCreateCampaignMemberInsert(){
         setupRelationshipTestData(null);
 
-        //reset the process controller to mimic a second user action
-        //Relationships_ProcessControl.hasRun = false;
-
         //Insert a campaign
-        Campaign campaign = new Campaign(Name='TestCampaign', isActive=true, Type='Other');
+        Campaign campaign = new Campaign(Name='TestCampaign', IsActive=true, Type='Other');
         insert campaign;
 
         //Insert a campaign member
-        CampaignMember cm = new CampaignMember(ContactId = c2.id, CampaignId = campaign.id);
+        CampaignMember cm = new CampaignMember(ContactId = c2.Id, CampaignId = campaign.Id);
```

Comment:
more minor stuff here, but for arguments in a method, (don't ask me why) I never use spaces for variable assignment, (like line 221), rather than having spaces (line 225). Just trying to make it more consistent.

Reply:
I agree we should be consistent. Why don't you like spaces here but do like spaces in other areas? (like WHERE clauses?)

Reply:
I REALLY wish I had a good answer for this, but I don't which is why I don't care *what* we do as long as we start doing it consistently.  Definitely not saying is has to be one way over another and recognize the irony in my waffling about consistency yet in actuality me suggesting this is kinda NOT consistent. Probably more of a discussion than an action item.